opentutor_classifier_api/Makefile

```diff
@@ -1,49 +1,19 @@
-BLACK_EXCLUDES="/(\.venv|build)/"
 DOCKER_IMAGE?=opentutor-classifier-api
-DOCKER_IMAGE_ID=$(shell docker images -q ${DOCKER_IMAGE} 2> /dev/null)
 ROOT=$(shell dirname ${PWD})
-OPENTUTOR_CLASSIFIER_SRC=$(ROOT)/opentutor_classifier
 VENV=.venv
-VENV_PIP=$(VENV)/bin/pip
+
 $(VENV):
-	$(MAKE) $(VENV)-update
-
-.PHONY $(VENV)-installed:
-$(VENV)-installed:
-	$(ROOT)/tools/virtualenv_ensure_installed.sh
-
-.PHONY: $(VENV)-update
-$(VENV)-update: $(VENV)-installed
-	[ -d $(VENV) ] || virtualenv -p python3.8 $(VENV)
-	$(VENV_PIP) install --upgrade pip
-	$(VENV_PIP) install -r requirements.test.txt
-	$(VENV_PIP) install ../opentutor_classifier
-	$(VENV)/bin/python3.8 -m nltk.downloader punkt
-	$(VENV)/bin/python3.8 -m nltk.downloader wordnet
-	$(VENV)/bin/python3.8 -m nltk.downloader averaged_perceptron_tagger
-	$(VENV)/bin/python3.8 -m nltk.downloader stopwords
+	$(MAKE) install
 
 .PHONY clean:
 clean:
-	rm -rf .pytest_cache .mypy_cache build
-
-build/opentutor_classifier:
-	mkdir -p build
-	rsync -rv \
-			--exclude .venv \
-			--exclude htmlcov \
-			--exclude .pytest_cache \
-			--exclude tests \
-			--exclude __pycache__ \
-		$(OPENTUTOR_CLASSIFIER_SRC) build
-
+	rm -rf .pytest_cache .mypy_cache
```

Comment:
let's change `clean` to also delete poetry's VENV, e.g

```python
clean:
	rm -rf .pytest_cache .mypy_cache $(VENV)
```

...and do the same in all other Makefiles that use poetry VENV
```